docs/_panels/jenkins-nodes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Jenkins Nodes
+description: Information about Jenkins Nodes
+author: Bitergia
+screenshot: sigils/jenkins-nodes.png
+created_at: 2020-03-11
+grimoirelab_version: 0.2.38
+layout: panel
+---
+
+This panel provides an overview of the nodes used in the Jenkins instances.
+
+## Metrics
+
+From left to right and top to bottom, the metrics provided are:
+
+* **Summary**: total numbers of builds, jobs, nodes and build durations.
+* **Results**: a pie chart that summarizes the proportions of the build results (success, failure, unstable, aborted).
+* **Jenkins projects**: a pie chart that highlights the builds by project.
```

Comment:
I suggest counting nodes instead of builds, as the dashboard is based on nodes. We could rename the widget to `Projects and Number of Nodes`.